blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/analytics/KeenConfiguration.java

```diff
@@ -35,12 +35,12 @@ KeenProject project() {
     public static class DefaultKeenConfiguration extends KeenConfiguration {
         @Override
         public String getProjectId() {
-            return "595421390ee24f5b59032f79";
+            return "5950934f3d5e150f5ab9d7be";
```

Comment:
project id and key can change in future as well? key might rotate in future? In that case all the previous instances will stop emitting events, no?

Reply:
Thats right. Not sure of a good way of solving this yet. Thoughts?

Reply:
Maybe if there is error returned (403/401) then check for new values from jenkins.io or from public s3 bucket?

Other issue, the keen.io credentials are known so the analytics data is prone to getting spammed by malicious users. Not sure what we can do in this case. Its easier to do such tings for saas services.